contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Understanding page load time metrics in performance monitoring
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-12
+featuredImage: todo
+tags: []
+---
+
+Waiting for slow websites is like watching paint dry. It's the bane of productivity, the destroyer of efficiency, and the enemy of progress.
+
+In this tutorial, we'll show you how to understand different metrics on how quickly your pages load and how to identify opportunities for improvement using Posthog's [performance monitoring in session recordings tool](https://posthog.com/blog/posthog-changelog#performance-monitoring-in-session-recordings).
```

Comment:
Could simplify language here - e.g. we'll show you:
- The three most important metrics for measuring page load speed
- A cheat sheet to help you prioritize and improve them